brands/ulefone/README.md

```diff
@@ -0,0 +1,14 @@
+# Ulefone 
+
+* Verdict **ℹ️ "Safe for now" :trollface:**
+* [**🔓️ Unlock Guide (MediaTek)**](/misc/generic-unlock.md)
+* [**🔓️ Unlock Guide (Unisoc)**][Unisoc Unlock]
+
+
+Ulefone follows the [standard unlock procedure](/misc/general-unlock.md) for their MediaTek devices, and the [Unisoc procedure][Unisoc Unlock] for their Unisoc devices. However, similarly to Samsung devices, you have to run an extra command to [disable verification][Ulefone verification], or else the device will bootloop upon flashing any partition. You can easily recover from the bootloop by reflashing the stock partition, however. 
```

Comment:
> [disable verification][Ulefone verification]

If this is about `fastboot flash vbmeta_a --disable-verity --disable-verification vbmeta.img`, then it's a standard thing you have to do on basically all modern AVB devices.

Reply:
ah alright, i guess i only own weird devices then because i never had to do that